File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("heigh: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int j = n - 1; j > i; j--)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/p2_mario/mario.py
from cs50 import get_int

def main():

    while True:
        n = get_int("height: ")
        if n > 0 and n < 9:
            break
    for i in range(n):
        print(" " * (n - 1 - i), end = "")
        for j in range(i + 1):
            print("#", end = "")
        print("  ", end = "")
        for k in range(i + 1):
            print("#", end = "")
        print()


main()

# check50 cs50/problems/2021/x/sentimental/mario/more
# submit50 cs50/problems/2020/x/sentimental/mario/more